### DISCUSSION TITLE
I use CustomRememberMeAuthenticator Is it possible to remove part with FQN in cookie?

### DISCUSSION THREAD

**Discussion by GrishaKhoroshilov** [symfony/symfony#53804] [Q&A]
I use CustomRememberMeAuthenticator when the cookie looks like App.Modules.User.User.Domain.Entities.User%3AZy5ob3Jvc2hpbG92QG5mcGktZWR1LnJ1%3A1707740642%3AEyv4rnFJNjs5nLMMR60S1jLVoGpd-X_SpihpOhAlvmM~ wCkOrT2GqfVNo_2bT49YOS0NQxXqAll4atT4--roQAk~
I'm not happy with the left part, which essentially displays the namespace of the user class. Is it possible to remove this part?
Symfony 6.4
security.yaml
providers:

        main_user_provider:
            entity:
                class: App\Modules\User\User\Domain\Entities\User
                property: email

firewalls:

        main:
            lazy: true
            provider: main_user_provider
            user_checker: App\Security\UserChecker
            custom_authenticators:
                - App\Security\LoginFormCustomAuthenticator
                - App\Security\CustomRememberMeAuthenticator
            logout:
                path: /logout
                # where to redirect after logout
                target: app_login

            remember_me:
                secret:   '%kernel.secret%' # required
                lifetime: 604800 # 1 week in seconds
                signature_properties: ['password', 'email']

**Comment by MatTheCat** [comment]
Not yet; see https://github.com/symfony/symfony/issues/42349
